File: cpp/25c_1742A_Sum.cpp
#include<bits/stdc++.h>

int main(){
    std::vector<std::string> ans;
    int temp, t;
    std::cin>>t;
    for(int j = 0; j < t; j++){
        std::vector<int> numbs;    
        for(int i = 0; i < 3; i++){
            std::cin>>temp;
            numbs.push_back(temp);
        }
        sort(numbs.begin(), numbs.end());
        if(numbs[0] + numbs[1] == numbs[2]){
            ans.push_back("YES");
        }
        else{
            ans.push_back("NO");
        }
    }
    for(auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;   
}

File: python/25c_1742A_Sum.py
def main():
    t = int(input())
    ans = []
    for _ in range(t):
        numbs = input().split()
        for i in range(3):
            numbs[i] = int(numbs[i])
        numbs.sort()
        if numbs[0] + numbs[1] == numbs[2]:
            ans.append("YES")
        else:
            ans.append("NO")
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()